daemon/cluster/cluster.go

```diff
@@ -51,6 +51,7 @@ import (
 	types "github.com/docker/docker/api/types/swarm"
 	executorpkg "github.com/docker/docker/daemon/cluster/executor"
 	"github.com/docker/docker/pkg/signal"
+	"github.com/docker/libnetwork/cluster"
```

Comment:
IMO, I don't think there's a need to rename the package itself - it's common and expected for different packages to have colliding final components. However it seems like a good idea to import it as `lncluster` for clarity.